sycl/test-e2e/FilterSelector/select_device_opencl.cpp

```diff
@@ -9,7 +9,7 @@
 // REQUIRES: opencl,gpu,cpu,accelerator
 
 #include <iostream>
-#include <sycl/sycl.hpp>
+#include <sycl/detail/core.hpp>
```

Comment:
I think you missed a newline here, to be consistent the rest of files.

Reply:
I don't want to run CI for this. I think it will be fine like this.